### PR TITLE
Overall Survival Months clinical attribute in patient view precision

### PR DESCRIPTION
**Pull request by adamabeshouse**
Now has no decimal digit precision

# Checks
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

### CODE REVIEW

File: src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx [hunk 1]
```diff
@@ -21,6 +21,14 @@ interface IPatientRow {
 
 export default class ClinicalInformationPatientTable extends React.Component<IClinicalInformationPatientTableProps, {}> {
 
+    private renderValue(attribute:string, value:string):string {
+        if (attribute === "Overall Survival (Months)") {
+            return parseInt(value, 10).toFixed(0) + '';
```
Comment (alisman) [inline src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx]: what does addition of empty string at the end do?  toString?

Comment (onursumer) [inline src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx]: Seems like `toFixed` already returns a string value...

Comment (adamabeshouse) [inline src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx]: @onursumer yes of course...how else could it work :man_facepalming:

Comment (onursumer) [inline src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx]: So, we don't need to append an empty string to the end, do we?

**Pull request merged**